### ISSUE TITLE
Profile Page: public/private toggle tooltip more concise wording

### ISSUE THREAD

**Issue by bdolly** [kids-first/kf-portal-ui#2270]
Remove the `...`, as a user I would feel more secure giving and allowing you to share my information with others if I knew _exactly_ what information you were sharing and how. The user base for the portal is very data security literate and will likely be cautious with any data-sharing issues. 

![Screen Shot 2020-01-21 at 12 27 38 PM](https://user-images.githubusercontent.com/1334131/72827857-8d5a3580-3c49-11ea-9213-edd6db092836.png)

**Comment by evans-g-crsj** [comment]
good point
